spec/features/hub/clients_searching_sorting_and_filtering_spec.rb

```diff
@@ -57,6 +59,16 @@
           expect(page.all('.client-row').length).to eq 2 # make sure filter is retained
           expect(page.all('.client-row')[1]).to have_text(patty_prep_ready_for_call.preferred_name)
           expect(page.all('.client-row')[0]).to have_text(zach_prep_ready_for_call.preferred_name)
+
+          click_link "sort-primary_consented_to_service_at"
```

Comment:
This is a good test!

One day I think it'd be splendid to move this test to a unit/form/controller spec of some kind. It is fine for this PR not to be blocked on a bigger refactor. Getting things out the door is super important.